topics/single-cell/tutorials/bulk-deconvolution-evaluate/tutorial.md

```diff
@@ -0,0 +1,585 @@
+    - elixir-fair-data
+
+follow_up_training:
+  -
+    type: "internal"
+    topic_name: single-cell
+
+---
+
+There are various methods to estimate the proportions of cell types in bulk RNA data. Since the actual cell proportions of the data are unknown, how do we know if our tools are producing accurate results?
+
+In this tutorial we will be using single-cell data with known cell-type proportions in order to create pseudo-bulk RNA data. We will then estimate the cell-type proportions of this pseudo-bulk data using the currently available deconvolution tools within Galaxy. Since we know the true proportion values, we will be able to measure and compare the accuracy of the tools' predictions.
+
+> <agenda-title></agenda-title>
+>
+> In this tutorial, we will cover:
+>
+> 1. TOC
+> {:toc}
+>
+{: .agenda}
+
+# Get the single-cell data
+
+First we need to create a new history in Galaxy and load in our single-cell data. We are going to use the single-cell dataset from a previous deconvolution tutorial found here: [https://training.galaxyproject.org/training-material/topics/single-cell/tutorials/bulk-music/tutorial.html]({% link topics/single-cell/tutorials/bulk-music/tutorial.md %}).
+
+> <hands-on-title>Data upload</hands-on-title>
+>
+> 1. Create a new history for this tutorial *"Deconvolution: Evaluating Reference Data"*
+> 2. Import the files from [Zenodo](https://zenodo.org/records/5719228) or from
+>    the shared data library (`GTN - Material` -> `{{ page.topic_name }}`
+>     -> `{{ page.title }}`):
+>
+>    * Human pancreas single-cell RNA datasets (tag: `#scrna`)
+>      ```
+>      https://zenodo.org/record/5719228/files/EMTABesethealthy.expression.tabular
+>      https://zenodo.org/record/5719228/files/EMTABesethealthy.phenotype.tabular
+>      ```
+>
+>    {% snippet faqs/galaxy/datasets_import_via_link.md %}
+>
+>    {% snippet faqs/galaxy/datasets_import_from_data_library.md %}
+>
+> 4. Check the datatypes are `tabular`
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+>
+> 5. Add a `#metadata` tag to `EMTABesethealthy.phenotype.tabular` and a `#expression` tag to `EMTABesethealthy.expression.tabular`
+>
+>    {% snippet faqs/galaxy/datasets_add_tag.md %}
+>
+{: .hands_on}
+
+## Inspecting the single-cell data
+
+Before continuing lets quickly inspect our single-cell data. We can find all of the cell types present in the data alongside their proportions by using the count tool to count the occurrence of each cell type category in the metadata file.
+
+> <hands-on-title>Get cell counts</hands-on-title>
+>
+> 1. {% tool [Count](Count1) %} with the following parameters:
+>    - {% icon param-file %} *"from dataset"*: `EMTABesethealthy.phenotype.tabular`
+>    - *"Count occurrences of values in column(s)"*: `Column 5`
+>    - *"Delimited by"*: `Tab`
+>    - *"How should the results be sorted?"*: `With the most common value first`
+>
+> 2. **Rename** {% icon galaxy-pencil %} output `Cell type counts`

```

Comment:
maybe add the faq for renaming a dataset here at the first time people are asked to do it?